support-frontend/assets/helpers/abTests/mvt.ts

```diff
@@ -0,0 +1,22 @@
+import * as cookie from '../storage/cookie';
+
+const MVT_COOKIE = 'GU_mvt_id';
+const MVT_MAX = 1_000_000;
+
+// Attempts to retrieve the MVT id from a cookie, or sets it.
+export function getMvtId(): number {
```

Comment:
moved out of abtest.ts for reuse

Reply:
It looks like this is still in `abtest.ts` - should it be removed?

Reply:
good spot - removed now